components/brave_rewards/browser/content_browser_client_helper.h

```diff
@@ -0,0 +1,94 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
not sure why it got named this way in webtorrent (content_browser_client_helper), but it should rewards_protocol_handler. As mentioned in DM I'm also concerned about adding a new scheme

Reply:
done